src/components/Reviews.jsx

```diff
@@ -19,28 +21,54 @@ export default function Reviews() {
                 const params = new URLSearchParams(location.search);
                 const category = params.get('category');
                 if(category) {
-                    url += `?category=${category}`;
+                    url += `?category=${category}&sortBy=${sortBy}&order=${order}`;
+                } else {
+                    url += `?sortBy=${sortBy}&order=${order}`;
                 }
                 const response = await axios.get(url);
```

Comment:
You should have this logic abstracted in with your util functions and I would look at axios for the params object it can take as an argument